src/generate/overrides.js

```diff
@@ -0,0 +1,49 @@
+'use strict'
+
+// Source data is not perfect, below are manual overrides applied to geo names
+const overrides = new Map([
+  // ['original', 'override']
+  ['Antigua And Barbuda', 'Antigua and Barbuda'],
+  ['Bosnia & Herzegovina', 'Bosnia and Herzegovina'],
+  ['Bolivia, Plurinational State Of', 'Bolivia'],
+  ['Bonaire, Saint Eustatius And Saba', 'Bonaire, Saint Eustatius and Saba'],
+  ['Democratic Republic Of Congo', 'Democratic Republic of Congo'],
+  ['Republic Of Congo', 'Republic of Congo'],
+  ['Czech Republic', 'Czechia'],
+  ['Ceuta, Mulilla', 'Ceuta and Mulilla'],
+  ['Micronesia, Federated States Of', 'Micronesia'],
+  ['France, Metropolitan', 'Metropolitan France'],
+  ['South Georgia And The South Sandwich Islands', 'South Georgia and the South Sandwich Islands'],
+  ['Guinea-bissau', 'Guinea-Bissau'],
+  ['Heard Island And McDonald Islands', 'Heard Island and McDonald Islands'],
+  ['Isle Of Man', 'Isle of Man'],
+  ['Iran, Islamic Republic Of', 'Iran'],
+  ['Saint Kitts And Nevis', 'Saint Kitts and Nevis'],
+  ["Korea, Democratic People's Republic Of", 'DPR Korea'],
+  ['Korea, Republic Of', 'Republic of Korea'],
```

Comment:
Should this be `South Korea`?

Reply:
If we're generally dropping official names (e.g. Iran, Venezuela), slapping an official name here seems to usher in political nuance...